modules/ROOT/partials/configuration/model.adoc

```diff
@@ -0,0 +1,20 @@
+[[model]]
+== `+model+`
+
+This option allows you to specify the model that {productname} should use. The default model included with {productname} is *dom*. Custom models are not supported at this time.
```

Comment:
`you` should be avoided... a lot of old docs use it still but new docs shouldn't. See https://www.tiny.cloud/docs/configure/contributing-docs/#pronouns

```suggestion
This option allows configuring which model {productname} should use. The default model included with {productname} is *dom*. Custom models are not supported at this time.
```

Reply:
ok, this whole thing is 90% the `theme.adoc` file with `theme` replaced by `model`.

Reply:
Yeah I figured that was probably the case.